app/controllers/reportings_controller.rb

```diff
@@ -197,18 +199,7 @@ def new
   end
 
   def create
-    @reporting = @project.reportings_via_source.build
-    @reporting.reporting_to_project_id = params['reporting']['reporting_to_project_id']
-
-    if @reporting.reporting_to_project.nil?
-      flash.now[:error] = l('timelines.reporting_could_not_be_saved')
-      render action: :new, status: :unprocessable_entity
```

Comment:
You introduced a behavioral change here:

Before we would respond with an **HTTP 422** if the `reporting_to_project` was missing.
Now we will respond with whatever is the default for `render action: 'new'` (I guess HTTP 200?)